src/backend/api/Fusion.Resources.Domain/Queries/GetResourceAllocationRequests.cs

```diff
@@ -178,6 +178,7 @@ public async Task<QueryRangedList<QueryResourceAllocationRequest>> Handle(GetRes
                         m.MapField("state", i => i.State.State);
                         m.MapField("state.isComplete", i => i.State.IsCompleted);
                         m.MapField("provisioningStatus.state", i => i.ProvisioningStatus.State);
+                        m.MapField(nameof(QueryResourceAllocationRequest.ProposedPerson), x => x.ProposedPerson.AzureUniqueId);
```

Comment:
since you are mapping azure uniqueid, you should probably add proposedperson.azureuniqueid instead ?
or?